lib/model/QubitInformationObject.php

```diff
@@ -440,6 +440,11 @@ public static function getUpdatedRecords($options = array())
       $criteria = QubitAcl::addFilterDraftsCriteria($criteria);
     }
 
+    if (isset($options['topLevel']) && $options['topLevel'])
```

Comment:
Nitpick... wouldn't be more correct to write: `if (isset($options['topLevel']) && $options['topLevel'])`? Or do you really want to support any truthy value?